ui/component/lastReleaseChanges/view.jsx

```diff
@@ -35,7 +36,7 @@ const LastReleaseChanges = (props: Props) => {
   );
 
   useEffect(() => {
-    const lastReleaseUrl = 'https://api.github.com/repos/lbryio/lbry-desktop/releases/latest';
+    const lastReleaseUrl = `https://api.github.com/repos/lbryio/lbry-desktop/releases/tags/${releaseVersion}`;
```

Comment:
There's also something strange that happened - I pushed a tag with 'alpha' in it which ended up on top of the list, but there were no builds associated with it, so updates were failing. Probably some library code that just gets the last tag with a prerelease keyword in it. After I deleted the bad tag, it was fine.